### DISCUSSION TITLE
Need info on LocalQueryRunner , DistributedQueryRunner and KafkaQueryRunner

### DISCUSSION THREAD

**Discussion by namratas-trianz** [trinodb/trino#14659] [General]
Hi,  I am looking for some info on LocalQueryRunner ,DistributedQueryRunner and KafkaQueryRunner. 
I am using trino libraries to connect to Kafka and fetch data. Currently I am using LocalQueryRunner.
I do see these are in trino.testing packages. Is there any specific reason it is in testing package? Can you also please help to understand the difference between these.

**Comment by ebyhr** [comment]
@lukasz-stec already answered in https://trinodb.slack.com/archives/CP1MUNEUX/p1665983976255519

> QueryRunner is basically testing infrastructure to run trino cluster.
LocalQueryRunner is a single node cluster and DistributedQueryRunner contains multiple (fake) node and supports “distributed” queries.
KafkaQueryRunner uses DistributedQueryRunnerwith kafka connector configured with test kafka instance (run in a container).
I don’t think this was designed to be used as kind of kafka client. (edit
